<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QWidget" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QWidget" name="mainWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>514</width>
           <height>614</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>514</width>
           <height>614</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="mainLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="titleBar" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#titleBar{

background-color: rgb(251, 251, 251);}</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="topMargin">
                  <number>5</number>
                 </property>
                 <property name="rightMargin">
                  <number>10</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Button_close">
                   <property name="minimumSize">
                    <size>
                     <width>18</width>
                     <height>18</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>18</width>
                     <height>18</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{background-color:#c2c2c2;border-radius:9px}
QPushButton:hover {background-color:#cb4042;border-radius:9px;}
QPushButton:pressed {background-color:#ab3b3a;border-radius:9px;}</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="mainDisplayWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <widget class="QGraphicsView" name="background">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>-60</y>
                 <width>521</width>
                 <height>651</height>
                </rect>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: url(:/icons/luminadeer.jpg);</string>
               </property>
              </widget>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>40</y>
                 <width>511</width>
                 <height>381</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_login">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>400</width>
                      <height>100</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>字体传奇特战体-免费商用</family>
                      <pointsize>24</pointsize>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgba(120, 200, 250, 220);</string>
                    </property>
                    <property name="text">
                     <string>单词消除游戏</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="spacing">
                   <number>10</number>
                  </property>
                  <property name="leftMargin">
                   <number>70</number>
                  </property>
                  <property name="rightMargin">
                   <number>70</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_username">
                    <property name="font">
                     <font>
                      <family>Microsoft YaHei UI</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>用户名:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QtMaterialAutoComplete" name="lineEdit_username">
                    <property name="tabletTracking">
                     <bool>false</bool>
                    </property>
                    <property name="autoFillBackground">
                     <bool>false</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="inputMask">
                     <string/>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="maxLength">
                     <number>32767</number>
                    </property>
                    <property name="placeholderText">
                     <string/>
                    </property>
                    <property name="clearButtonEnabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>10</number>
                  </property>
                  <property name="leftMargin">
                   <number>70</number>
                  </property>
                  <property name="rightMargin">
                   <number>70</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_password">
                    <property name="font">
                     <font>
                      <family>Microsoft YaHei UI</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>  密码 :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QtMaterialAutoComplete" name="lineEdit_password">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="clearButtonEnabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <property name="spacing">
                   <number>30</number>
                  </property>
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QtMaterialRadioButton" name="radioButton_challenger">
                    <property name="font">
                     <font>
                      <family>Microsoft YaHei UI Light</family>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="mouseTracking">
                     <bool>true</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>闯关者</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QtMaterialRadioButton" name="radioButton_questioner">
                    <property name="font">
                     <font>
                      <family>Microsoft YaHei UI Light</family>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="mouseTracking">
                     <bool>true</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>出题者</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="spacing">
                   <number>100</number>
                  </property>
                  <property name="leftMargin">
                   <number>100</number>
                  </property>
                  <property name="rightMargin">
                   <number>100</number>
                  </property>
                  <item>
                   <widget class="QtMaterialFlatButton" name="pushButton_login">
                    <property name="font">
                     <font>
                      <family>Microsoft YaHei UI</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="mouseTracking">
                     <bool>true</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>登录</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QtMaterialFlatButton" name="pushButton_regist">
                    <property name="font">
                     <font>
                      <family>Microsoft YaHei UI</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="mouseTracking">
                     <bool>true</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>注册</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialAutoComplete</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialautocomplete.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialflatbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialRadioButton</class>
   <extends>QRadioButton</extends>
   <header location="global">qtmaterialradiobutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
